config/application.rb

```diff
@@ -37,5 +37,12 @@ class Application < Rails::Application
       g.jbuilder          false
       g.scaffold_stylesheet false
     end
+
+    # Permitted locales available for the application
+    I18n.available_locales = [:ar, :cs, :en, :es, :de, :fr, :pt, :th]
```

Comment:
Would love to pull this list from the one committed in the yml rather than this list, but wasn't sure how. Any suggestions? And/or would you prefer to limit language support list? (I wouldn't, esp for communities that have large immigrant populations.)